### DISCUSSION TITLE
How to access BPF_MAP_TYPE_LRU_PERCPU_HASH in cilium ebpf?

### DISCUSSION THREAD

**Discussion by kawhicurry** [cilium/ebpf#1237] [Q&A]
`Iterator` always return false while `Lookup` return error:
```
cpu 0: unexpected EOF
```
Not support yet?

**Comment by lmb** [comment]
That might be, the PerCPU LRU maps are kind of peculiar beasts. Feel free to open a bug report with a reproducer.

**Reply by kawhicurry** [reply]
Got it. I'll prepare the example code for this first.

**Comment by kawhicurry** [comment]
I try to reproduce the failing code but failed.

I'm not familiar with the ebpf map two weeks ago. I might have misunderstood the `PERCPU_LRU_HASH` and tried to treat it as `hash map in array`, so I use an array to extract the contents from hash map. I should take it as a normal `hash map` that return `array`.

Sorry for misreport that. I'll be careful next time.
